Tests/Tests/RouterSpec.swift

```diff
@@ -217,6 +259,52 @@ class RouterSpec: QuickSpec {
                 }
             }
 
+            context("when calling isDisplayingTooltip") {
+
+                let tooltipTargetView = UIView(frame: CGRect(x: 100, y: 100, width: 10, height: 10))
+                let tooltip = TestHelpers.generateTooltip(id: "test")
+                let imageBlob: Data! = UIImage(named: "test-image", in: .unitTests, with: nil)?.pngData()
+
+                beforeEach {
+                    tooltipTargetView.accessibilityIdentifier = TooltipViewIdentifierMock
+                    window.addSubview(tooltipTargetView)
+                }
+
+                afterEach {
+                    tooltipTargetView.removeFromSuperview()
+                    window.findTooltipView()?.removeFromSuperview()
+                }
+
+                it("will return true when tooltip displayed") {
+                    router.displayTooltip(tooltip, targetView: tooltipTargetView,
+                                          identifier: TooltipViewIdentifierMock,
+                                          imageBlob: imageBlob,
+                                          becameVisibleHandler: { _ in },
+                                          confirmation: true,
+                                          completion: { _ in })
+                    expect(window.findTooltipView()).toEventuallyNot(beNil())
+                    expect(router.isDisplayingTooltip(with: TooltipViewIdentifierMock)).to(beTrue())
+                }
+
+                it("will return false when tooltip is not displayed") {
+                    expect(router.isDisplayingTooltip(with: "test")).to(beFalse())
+                }
+
+                it("will return false when tooltip identifier is incorrect") {
+                    expect(router.isDisplayingTooltip(with: "test")).to(beFalse())
```

Comment:
But you don't display any tooltip here.
You need to have a code like this before the final expectation:
```
                    router.displayTooltip(tooltip, targetView: tooltipTargetView,
                                          identifier: TooltipViewIdentifierMock,
                                          imageBlob: imageBlob,
                                          becameVisibleHandler: { _ in },
                                          confirmation: true,
                                          completion: { _ in })
                    expect(window.findTooltipView()).toEventuallyNot(beNil())
```

Reply:
Updated here [72da23d](https://github.com/rakutentech/ios-inappmessaging/pull/275/commits/72da23dfdd6189fdffe59dbeb76b66e340dc6f9c).